app/routes/account.$accountId.$appId/view.tsx

```diff
@@ -210,6 +210,12 @@ export default function AppIdLayoutView({
   const isMember = role === RoleName.Member
   const isAdmin = role === RoleName.Admin
 
+  console.log(
```

Comment:
haha, what is this?

Reply:
Damn I didn't know I pushed this 😂
I got so many errors and I wanted to easily see my logged message